includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -3,48 +3,81 @@
  * AMP setup wizard page.
  *
  * @package AMP
- * @since @todo NEW_ONBOARDING_RELEASE_VERSION
```

Comment:
This `todo` should be updated in the `Test_AMP_Setup_Wizard_Submenu_Page` and `AMP_Setup_Wizard_Submenu` classes as well.